lisa/node.py

```diff
@@ -252,6 +252,8 @@ def cleanup(self) -> None:
         if hasattr(self, "_log_handler") and self._log_handler:
             remove_handler(self._log_handler, self.log)
             self._log_handler.close()
+        if hasattr(self, "first_initialize"):
```

Comment:
1. The log_handler is to save two states (first time, initialized), so the getattr is used to check it. the first_initialize doesn't need. So just set it to false in `__init__`.
2. The name should be `_first_initialize`, because it's a private value.

Reply:
updated